website/content/en/docs/tasks/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Tasks
+description: Tasks and guidelines for the ClusterLink system.
```

Comment:
what did you have in mind for `guidelines`?
I would also drop "system" - I think we should be using ClusterLink as a noun, without "system" or "project" added.